src/scss/objects/_form-toggle.scss

```diff
@@ -13,18 +13,18 @@
  */
 
 $formToggle-padding                 : spacing() !default;
-$formToggle-border-color            : $grey--lightest;
-$formToggle-border-color-interact   : $grey--lighter;
+$formToggle-border-color            : $grey--lighter;
+$formToggle-border-color-interact   : $grey--light;
 $formToggle-border-color-checked    : $green;
 $formToggle-border-width            : 1px;
 $formToggle-border-radius           : 8px;
 $formToggle-icon-background         : $green;
-$formToggle-icon-hover-background   : $grey--lighter;
+$formToggle-icon-hover-background   : $grey--light;
 $formToggle-background              : $white;
 $formToggle-large-size              : 48px;
 $formToggle-button-color            : $blue;
 $formToggle-button-background       : $blue--offWhite;
```

Comment:
`$btn-secondary-bgColor`?

Reply:
I'll keep these separate for now. When the PIE aliases come in, it might be that we have a wider form control background and I'll update them to that.